product/scanning.py

```diff
@@ -40,6 +39,7 @@ def __init__(self, protecode_api: ProtecodeApi, group_id=None):
         protecode_api.login()
         self._api = not_none(protecode_api)
         self._group_id = group_id
+        self._last_analysis_result = None
```

Comment:
this makes the PCU thread-unsafe (but it is used in a multithreading ctx)
--> rm the field and pass the value via method-args